### PR TITLE
11.0 hr views

### PR DESCRIPTION
**Pull request by jarroyomorales**
Travis

### CODE REVIEW

**Comment by etobella** [comment]
The res_partner files are missing, try to add them please.

File: cb_hr_views/data/ir_sequence_data.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<odoo>
+    <data noupdate="0">
+        <record id="seq_hr_employee" model="ir.sequence">
+            <field name="name">Employee sequence</field>
+            <field name="code">hr.employee</field>
+        </record>
+    </data>
+</odoo>
```
Comment (etobella) [inline cb_hr_views/data/ir_sequence_data.xml]: Intro

File: cb_hr_views/security/hr_attendance_security.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    <record id="hr_attendance.group_hr_attendance" model="res.groups">
+        <field name="name">Employee</field>
+    </record>
+
+    <record id="hr_attendance.hr_attendance_rule_attendance_manager" model="ir.rule">
+            <field name="active" eval="False"/>
+    </record>
+
+    <record id="hr_attendance_rule_attendance_employee" model="ir.rule">
```
Comment (etobella) [inline cb_hr_views/security/hr_attendance_security.xml]: All rules should have noupdate.

**Review by etobella — CHANGES_REQUESTED** [review]

File: cb_hr_views/models/hr_employee.py [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        related='partner_id.firstname'
+    )
+    lastname = fields.Char(
+        related='partner_id.lastname'
+    )
+    lastname2 = fields.Char(
+        related='partner_id.lastname2'
+    )
+
+    active = fields.Boolean(
```
Comment (etobella) [inline cb_hr_views/models/hr_employee.py]: That is wrong, as active is already related to resource. You should modify the functions toggle_active on partner, resource and hr.employee in order to make it work at the same time.

**Review by etobella — CHANGES_REQUESTED** [review]

File: cb_hr_views/models/hr_employee.py (hunk 3 above)
Comment (jarroyomorales) [reply cb_hr_views/models/hr_employee.py]: and `res.users` too?

**Review by etobella — APPROVED** [review]

**Pull request merged**